app/controllers/api/databases.php

```diff
@@ -2033,12 +2122,13 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
         }
 
         $filterQueries = Query::groupByType($queries)['filters'];
+        $timeoutMilliseconds = 200; // todo: make this configurable
```

Comment:
```suggestion
        $timeoutMilliseconds = 200; // todo: make this configurable
```

Should be enough to just mention this unit in the doc block instead of using it as part of the name.

Reply:
Changes